uint8 drone_id